operators/benches/thread_pool.rs

```diff
@@ -0,0 +1,29 @@
+#![feature(bench_black_box)]
```

Comment:
We shouldn't start using more nightly features I guess, one step more apart from using stable.